src/routes/courses.js

```diff
@@ -1,8 +1,16 @@
 const express = require("express");
+const path = require("path");
+const fs = require("fs");
 const router = express.Router();
 
+const rootPath = path.join(__dirname, "../", "../");
+const coursesPath = path.join(rootPath, "data/courses.json");
```

Comment:
```suggestion
 const coursesPath = path.join(__dirname, "../data/courses.json");
```
and you can remove `rootPath`

Reply:
Done